src/03-modality-agnostic-files.md

```diff
@@ -14,6 +14,23 @@ Templates:
 The file `dataset_description.json` is a JSON file describing the dataset.
 Every dataset MUST include this file with the following fields:
 
+{{ MACROS___make_metadata_table(
+   {
+      "Name": "REQUIRED",
```

Comment:
I don't see name in rendered version: https://bids-specification--762.org.readthedocs.build/en/762/03-modality-agnostic-files.html#dataset_descriptionjson